common/utils/src/main/resources/error/error-classes.json

```
@@ -2256,6 +2256,12 @@
     ],
     "sqlState" : "42613"
   },
+  "INVALID_SAVE_MODE" : {
+    "message" : [
+      "The specified save mode '<mode>' is invalid. Valid save modes include 'append', 'overwrite', 'ignore', 'error', 'errorifexists', and 'default'."
```

Let's quite the option values by default "":
```suggestion
      "The specified save mode <mode> is invalid. Valid save modes include \"append\", \"overwrite\", \"ignore\", \"error\", \"errorifexists\", and \"default\"."
```